src/components/AppFooter.jsx

```diff
@@ -0,0 +1,106 @@
+import Link from "next/link";
```

Comment:
Why is there a .js and a .jsx file?

Reply:
Not sure, .js seems to be from an old version. I deleted it now.